CMakeLists.txt

```
@@ -1,4 +1,9 @@
 cmake_minimum_required(VERSION 3.23.1)
```

```suggestion
cmake_minimum_required(VERSION 3.26.3)
```

@CusiniM @wrtobin I think this is a good version to set...any objection?

---

Currently, the newest cmake available on Frontier via module load is `3.23.2`